Microsoft.Toolkit.Parsers/Markdown/Helpers/Common.cs

```diff
@@ -12,11 +12,14 @@
 
 using System.Collections.Generic;
 using System.Linq;
-using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Parse;
+using Microsoft.Toolkit.Parsers.Markdown.Parse;
 
-namespace Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Helpers
+namespace Microsoft.Toolkit.Parsers.Markdown.Helpers
 {
-    internal class Common
+    /// <summary>
+    /// Helpers for Markdown.
+    /// </summary>
+    public class Common
```

Comment:
any reason why this is public?

Reply:
It used to be an issue with creating a Parser for different Libraries, but it might not now.

Reply:
Fixed.